.github/actions/setup-triton/action.yml

```
@@ -99,6 +99,7 @@ runs:
       run: |
         cd python
         pip install wheel pybind11
+        pip install -U "setuptools>=78.1.0"
```

Per comment here https://github.com/intel/intel-xpu-backend-for-triton/issues/3872#issuecomment-2788334511
```suggestion
        pip install -r requirements.txt
```